samples/cpp/beam_search_causal_lm/beam_search_causal_lm.cpp

```diff
@@ -8,22 +8,21 @@ int main(int argc, char* argv[]) try {
         throw std::runtime_error(std::string{"Usage: "} + argv[0] + " <MODEL_DIR> '<PROMPT 1>' ['<PROMPT 2>' ...]");
     }
     auto prompts = std::vector<std::string>(argv + 2, argv + argc);
-    
     std::string model_path = argv[1];
-    std::string device = "CPU";  // GPU can be used as well
 
+    std::string device = "CPU";  // GPU can be used as well
     ov::genai::LLMPipeline pipe(model_path, device);
-    ov::genai::GenerationConfig config = pipe.get_generation_config();
+
+    ov::genai::GenerationConfig config;
     config.max_new_tokens = 20;
     config.num_beam_groups = 3;
     config.num_beams = 15;
     config.num_return_sequences = config.num_beams * prompts.size();
```

Comment:
I suppose the usage is incorrect. `num_return_sequences` is responsible for a single prompt. 
So, `num_return_sequences <= config.num_beams` should be always true